config/database.yml

```diff
@@ -12,7 +12,10 @@ default: &default
 
 development:
   <<: *default
-  database: siglateste
+  # database: siglateste
+  host: db
+  username: postgres
+  password:
```

Comment:
I'm using postgres as password to the database

Reply:
Hmm, I'll change it! Thanks for the tip!